examples/dreamcast/raylib/tetris/src/vmu/vmuManager.cpp

```diff
@@ -0,0 +1,20 @@
+#include "vmuManager.h"
+
+#include "../constants/vmuIcons.h"
+
+VmuManager::VmuManager(){
+    resetImage();
+}
+
+VmuManager::~VmuManager(){
+}
+
+void VmuManager::displayImage(const char *xmp){
+    maple_device_t * vmu = maple_enum_type(0, MAPLE_FUNC_LCD);
```

Comment:
```suggestion
    maple_device_t *vmu = maple_enum_type(0, MAPLE_FUNC_LCD);
```
Sorry... but the last thing I want the kiddos to learn is that horrible style NOBODY likes. ;)